<launch>
    <rosparam command="load" file="$(find fastlio2_sam_lc)/config/mapping.yaml" />
    <node pkg="fastlio2_sam_lc" type="map_builder_node" name="map_builder_node" output="screen"/>
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fastlio2_sam_lc)/rviz/mapping.rviz" />
</launch>